src/pagecache/mod.rs

```diff
@@ -59,16 +59,19 @@ pub use self::{
     segment::SegmentMode,
 };
 
-/// An offset for a storage file segment.
+/// The offset of a segment. This equals its LogOffset (or the offset of any
+/// item contained inside it) divided by the configured segment_size.
 pub type SegmentId = usize;
 
-/// A log file offset.
-pub type LogId = u64;
+/// A file offset in the database log.
+pub type LogOffset = u64;
 
 /// A pointer to an blob blob.
 pub type BlobPointer = Lsn;
 
-/// A logical sequence number.
+/// A logical sequence number of a log entry.
+/// A segment lsn equals to an aligned LogOffset that is
+/// Segment.LogOffset & ~SegmentSize.
```

Comment:
please no symbols or CamelCase names that don't match to real names in the code.